dashboard/app/models/levels/blockly.rb

```diff
@@ -398,4 +389,9 @@ def fix_examples
     return if examples.nil?
     self.examples = examples.select(&:present?)
   end
+
+  # This should just return false, but we have a few levels.js levels that use
+  # droplet (starwars, and a few test levels). They all have level records of
+  # type 'Blockly', so they can't override this as needed
+  delegate :uses_droplet?, to: :game
```

Comment:
Why delegate vs. moving the method from game.rb to here, if this is the only place it's used?

Reply:
I was on the fence. It feels like a method that belongs in the game class since it only depends on the game type, but yeah if it's not actually used there then I suppose it make more sense to move it to blockly.rb.